contentcuration/contentcuration/frontend/channelEdit/components/edit/__tests__/resourcesNeededOptions.spec.js

```diff
@@ -0,0 +1,33 @@
+import Vue from 'vue';
+import Vuetify from 'vuetify';
+import { shallowMount } from '@vue/test-utils';
+import ResourcesNeededOptions from '../ResourcesNeededOptions.vue';
+import { ResourcesNeededTypes } from 'shared/constants';
+
+Vue.use(Vuetify);
+
+describe('ResourcesNeededOptions', () => {
+  it('smoke test', () => {
+    const wrapper = shallowMount(ResourcesNeededOptions);
+
+    expect(wrapper.isVueInstance()).toBe(true);
+  });
+
+  it('when there is a list of keys to remove from ResourcesNeededTypes, return updated map for ResourcesNeededTypes for dropdown', () => {
```

Comment:
Would want to update this for a non-dynamic test - just ensure that the excluded keys are indeed excluded from the dropdown.